edge-hub/watchdog/linux/src/main.rs

```diff
@@ -0,0 +1,92 @@
+extern crate nix;
+
+// TODO: standardize imports
+use signal_hook::{iterator::Signals, SIGTERM};
+use std::{error::Error, thread};
+use std::process::{Command, Stdio, Child};
+use std::sync::{Arc};
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::time::Duration;
+use nix::unistd::Pid;
+use nix::sys::signal::{self, Signal};
+
+// TODO: should we set up logging more formally with dependency
+// TODO: if not, then we can add info / error
+const LOG_HEADER: &'static str = "[WATCHDOG]:";
```

Comment:
Should we use a dependency for logging? If not, it would be helpful to add INFO, ERROR labels.

Reply:
Also, it is logging the quotes for some reason.